controllers/follow.js

```diff
@@ -0,0 +1,90 @@
+import Sequelize from 'sequelize';
+import models from '../models';
+
+const { iLike } = Sequelize.Op;
+const { User, Follow, Profile } = models;
+
+export default {
+  async addFollower(req, res) {
+    const userToFollow = await User.findOne({
+      where: { username: { [iLike]: req.params.username } },
+      include: [
+        {
+          model: Profile,
+          attributes: ['lastName', 'firstName', 'bio', 'category', 'imageUrl']
+        }
+      ]
+    });
+    await Follow.create({
+      followerId: req.decoded.id,
+      userId: userToFollow.id
+    });
+    return res.json({
+      message: `You are now following ${userToFollow.username}`,
+      profile: {
+        username: userToFollow.username,
+        firstName: userToFollow.Profile.firstName,
+        lastName: userToFollow.Profile.lastName,
+        bio: userToFollow.Profile.bio
+      }
+    });
+  },
+  async getFollowing(req, res) {
```

Comment:
I love this and the controller that comes after it. Just one thing. Put a space between successive controllers.

Reply:
on It